packages/tabs/src/component-tab.tsx

```diff
@@ -1,32 +1,20 @@
 import React from 'react';
 import { classNames as cn } from '@chbphone55/classnames';
-import { tab as c } from '@warp-ds/component-classes';
+import { tab as ccTab } from '@warp-ds/component-classes';
 import type { TabProps } from './props';
 
-const setup = ({
-  className,
-  isActive,
-  setActive,
-  contained,
-  ...rest
-}: any) => ({
-  tab: cn({
+const setup = ({ className, isActive, setActive, ...rest }: any) => ({
+  tab: cn([ccTab.tab], {
```

Comment:
Minor, I think you can even skip the array here
```suggestion
  tab: cn(ccTab.tab, {
```